Podfile

```diff
@@ -15,16 +15,14 @@ if Gem::Version.new(Pod::VERSION) > Gem::Version.new('1.9.3')
   install! 'cocoapods', :warn_for_unused_master_specs_repo => false
 end
 
-
-
 target 'Sequel Ace' do
   # Comment the next line if you don't want to use dynamic frameworks
   # use_frameworks!
 
   # Pods for Sequel Ace
   pod 'FirebaseCore'
   pod 'Firebase/Crashlytics'
-
+  pod 'FMDB', :git => 'https://github.com/ccgus/fmdb.git'
```

Comment:
I would prefer to target a version rather than master

Reply:
yes sorry meant to target a tag

Reply:
Done.